docs/user_guide/email_notifications/index.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: documentation
+title: Email Notifications
+sidebar: sidebars/documentation.html
+---
+
+# Email Notifications
+
+## Types of Email Notifications
+
+In addition to the Email Notifications that Foreman provides:
+
+* Puppet run summary (Daily/Weekly/Monthly)
+* Puppet errors
+
+Katello provides a few addition reports:
+
+* Katello Host Advisory (Daily/Weekly/Monthly) - A report of all of the Errata applicable to all readable Content Hosts
+* Katello Promote Errata - A report generated at Content View promotion time showing what new Errata have been promoted and how many Content Hosts are applicable.
+* Katello Sync Errata - A report generated after each Repository sync listing new Errata synced and how many Content Hosts are applicable.
+
+## Configuring the Foreman/Katello to send emails:
+
+The configuration on how the Foreman/Katello service sends email is located in /etc/foreman/email.yaml:
+
+```
+production:
+   delivery_method: :smtp
+   smtp_settings:
+     address: smtp.example.com
+     port: 25
+     domain: example.com
+     authentication: :none
+```
+
+For more information see: [Email Configuration](http://projects.theforeman.org/projects/foreman/wiki/Email_configuration)
```

Comment:
This info was promoted to the manual: http://theforeman.org/manuals/1.7/index.html#3.5.2ConfigurationOptions